README.md

```diff
@@ -47,6 +48,13 @@ and don't forget to read our
 [Contribuitor's Guide](https://flusp.ime.usp.br/doc/kw/content/howtocontribute.html)
 (or [howtocontribute file](documentation/content/howtocontribute.rst)).
 
+# Reach Out
+
+You can reach out to us at the `#kw-devel` channel on the
+[OFTC](https://www.oftc.net/) IRC server (**irc.of</span>tc.n</span>et**), or
+by joining or opening a new discussion on the
+[discussions](https://github.com/kworkflow/kworkflow/discussions) page.
```

Comment:
Add something before this paragraph that highlights that using Issues and PR for discussion is the preferable way.